2022/_data/talks.json

```diff
@@ -713,7 +710,7 @@
       "id": "62505edfaa23f0085462dda4",
       "title": "The Microservices Smackdown: Data",
```

Comment:
we should translate this title so that people don't get confused...

Reply:
That's agreed with speaker, the title is a reference to the american show, so there's no translation

Reply:
ohh , of course! 😅😂